Source/WebCore/svg/SVGFEConvolveMatrixElement.cpp

```diff
@@ -25,6 +25,7 @@
 #include "DocumentInlines.h"
 #include "FEConvolveMatrix.h"
 #include "NodeName.h"
+#include "SVGDocumentExtensions.h"
```

Comment:
Are all the single extra inclused like this one requires? What changed?

Reply:
Adding new files caused compilation errors because of unified sources. Without including this header file, I get errors in compiling this line in this file:
```
protectedDocument()->checkedSVGExtensions()->reportWarning(...);
```